spec/models/validation_request_spec.rb

```diff
@@ -276,5 +276,57 @@
         end
       end
     end
+
+    describe "#auto_close_request!" do
+      let(:request) { create(:red_line_boundary_change_validation_request, :open) }
+
+      describe "when successful" do
+        it "auto closes the request and creates an audit record" do
+          expect { request.auto_close_request! }
+            .to change(request, :auto_closed_at).from(nil).to(Time.current)
+                                                .and change(request, :state).from("open").to("closed")
+                                                                            .and change(request, :auto_closed).from(false).to(true)
```

Comment:
This alignment looks a bit weird to me...was it a rubocop autocorrect?

Reply:
lol yes, rubocop on a mad one